### PR TITLE
SAPP-958: Add and install NewRelic agent and config

### PR DESCRIPTION
**Pull request by acbramley**
Adds and configures NewRelic PHP agent for s21 images.

Disabled by default.

The following environment variables will then be added via shepherd environment variables:

```
      NEW_RELIC_ENABLED: 'true'
      NEW_RELIC_LICENSE_KEY: xyz
      NEW_RELIC_APP_NAME: environment-name
```

### CODE REVIEW

**Comment by singularo** [comment]
Doh - https://github.com/universityofadelaide/s2i-shepherd-drupal/pull/19/files

File: Dockerfile [hunk 1]
```diff
@@ -56,6 +56,16 @@ RUN apt-get update \
...
 
+# NewRelic is disabled by default.
+ENV NEW_RELIC_ENABLED=false
+
+# Install NewRelic agent https://docs.newrelic.com/docs/agents/php-agent/installation/php-agent-installation-ubuntu-debian
+RUN echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | tee /etc/apt/sources.list.d/newrelic.list && \
+    wget -O- https://download.newrelic.com/548C16BF.gpg | apt-key add - && \
+    apt-get update && \
+    apt-get install -y newrelic-php5 && \
+    rm -f /etc/php/7.2/mods-available/newrelic.ini
```
Comment (singularo) [inline Dockerfile]: That will leave empty, dangling symlinks in:
/etc/php/7.2/apache2/conf.d/20-newrelic.ini & 
/etc/php/7.2/cli/conf.d/20-newrelic.ini
I think

Comment (acbramley) [reply Dockerfile]: Fixed!

File: files/newrelic.ini [hunk 2]
```diff
@@ -0,0 +1,858 @@
...
+; Type   : boolean
+; Scope  : per-directory
+; Default: true
+; Info   : Enable or disable the agent. Please note that you cannot globally
+;          disable the agent and then selectively enable it on a per-directory
+;          basis. If you disable the agent in the global INI file then the
+;          agent will not initialize at all. However, you can selectively
+;          disable the agent on a per-directory basis.
+;
+newrelic.enabled = "${NEW_RELIC_ENABLED}"
```
Comment (singularo) [inline files/newrelic.ini:23]: Not clear to me how we are going to get a value in here. ini files aren't evaluated before loading, and not modifyable once running.

Comment (acbramley) [reply files/newrelic.ini:23]: It'll get environment variables from the pod. See https://github.com/universityofadelaide/shepherd/pull/162

Comment (acbramley) [reply files/newrelic.ini:23]: You can test this locally as well by manually building this image and using it as the FROM in the local Dockerfile then adding the environment variables in the web pod.

Comment (singularo) [reply files/newrelic.ini:23]: https://www.php.net/manual/en/configuration.file.php#example-36 - So good!

**Comment by singularo** [comment]
Looks good to me.

**Pull request merged**